charts/apisix/values.yaml

```diff
@@ -448,6 +448,9 @@ apisix:
       # - name: bar
       #   size: 1m
 
+    # Set the size of prometheus-metrics shared dict
+    prometheusSharedDictSize: 15m
```

Comment:
🤔 Is it a good idea to use a separate orphaned key?

Wouldn't it be better to implement it as a map and allow the original shared dict name used by APISIX to be passed in directly for overriding. e.g.

```yaml
luaSharedDicts:
  - name: prometheus-metrics
    size: 100m
```
If you think this is unclear, perhaps it could also be called `builtinLuaSharedDicts`. Or, make it configurable by `customLuaSharedDicts`, although not intuitive.

BTW, if you update the values template, please update the documentation along with it.

Reply:
it is very specific (in the otherwise unused meta key) in the upstream config, and the same mechanism doesn't seem to be used anywhere else, so I opted for a single specific key